monetio/obs/ish_lite.py

```diff
@@ -127,31 +138,37 @@ def read_data_frame(self, file_object):
         frame = pd.DataFrame.from_records(frame_as_array)
         df = self._clean(frame)
```

Comment:
This is not defined. Maybe we could just remove `read_data_frame` and `get_url_file_objs` ?

Reply:
Yes we can remove the `read_data_frame` routine.  It's a carry over from the `ish` reader.

the `get_url_file_objs` allows for downloading the data raw data. if this isn't required we can remove it

Reply:
Left the latter for now.